File: ruby/gilded_rose_test.rb
require 'test/unit'
require_relative 'gilded_rose'


class GildedRoseTest <  Test::Unit::TestCase

  attr_reader :gilded_rose, :items, :random

  def setup
    @gilded_rose = GildedRose.new
    @items = gilded_rose.make_items
  end

  def test_after_one_day
    gilded_rose.update_quality(items)
    # assert_equal(items[0])
  end


end

File: java/src/test/java/org/joyofcoding/objectcalisthenics/GildedRoseTest.java
package org.joyofcoding.objectcalisthenics;

import java.util.ArrayList;
import java.util.List;
import java.util.Random;

import org.joyofcoding.objectcalisthenics.assertions.ItemsAssert;
import org.junit.Before;
import org.junit.Test;

public class GildedRoseTest {
    private GildedRose gildedRose;
    private List<Item> items;

    @Before
    public void setup() {
        gildedRose = new GildedRose();
        items = gildedRose.makeItems();
    }
    
    @Test public void 
    after_one_day() throws Exception {
        gildedRose.updateQuality(items);
        // Assertions.assertThat(items[0])...
    }

}
